src/rimage.c

```diff
@@ -193,7 +193,7 @@ int main(int argc, char *argv[])
 
 	ret = ext_man_write(&image);
 	if (ret < 0) {
-		fprintf(stderr, "warning: unable to write extended manifest, %d\n",
+		fprintf(stdout, "warning: unable to write extended manifest, %d\n",
```

Comment:
This should really only be printf if we ask rimage to generate the ext manifest. i.e. we have a rimage cmd line option for ext manifest generation. i.e.(using -x for ext manifest generation)
1) ```rimage -x``` -> emits error message and stops if ext man not found
2) ```rimage``` -> no error or warning if ext manifest not found.

Reply:
`-e` flag is used, because `-x` is already in use